docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -185,7 +185,7 @@ It was released as a beta feature in Grafana 6.7. The feedback has been really p
 
 ## Time zone support
 
-You can now override the [time zone]({{< relref "../reference/timerange/#time-zone-options" >}}) used to display date and time values in a dashboard. One benefit of this is that you can specify the local time zone of the service or system that you are monitoring which can be helpful when monitoring a system or service that operates across several time zones.
+You can now override the [time zone]({{< relref "../dashboards/time-range-controls.md" >}}) used to display date and time values in a dashboard. One benefit of this is that you can specify the local time zone of the service or system that you are monitoring which can be helpful when monitoring a system or service that operates across several time zones.
```

Comment:
Already a PR open to resolve this: https://github.com/grafana/grafana/pull/24794

Reply:
Yeah, but I can't get this one to pass the build tests unless I fix the link.